cfgmgr/vxlanmgr.cpp

```diff
@@ -130,6 +140,16 @@ static int cmdUpVxlanIf(const swss::VxlanMgr::VxlanInfo & info, std::string & re
     return swss::exec(cmd.str(), res);
 }
 
+static int cmdDownVxlanIf(const swss::VxlanMgr::VxlanInfo & info, std::string & res)
```

Comment:
Same as above. These APIs are called only for VNET table. Are these changes needed?